language/stdlib/modules/LibraAccount.move

```diff
@@ -562,6 +574,13 @@ module LibraAccount {
         );
         Event::publish_generator(&new_account);
         add_currencies_for_account<Token>(&new_account, add_all_currencies);
+        // VASPs start off by default with the full range of currencies. If a
+        // new currency is added to the system they will need to call
+        // `VASP::try_allow_currency<NewCurrency>(parent_account)` in order to
+        // allow any of their child VASP accounts to hold a balance of `NewCurrency`.
+        VASP::try_allow_currency<LBR>(&new_account);
```

Comment:
Could assert the return value of `try_allow_currency` for extra assurance? Otherwise, these checks could be erroneously moved before `publish_parent_vasp_credential` and nothing would crash (I think).

Reply:
ah! good point. I think I can remove these if we say that a parent vasp needs to explicitly allow a currency before it can be used.